packages/plugin-vite/src/index.ts

```diff
@@ -60,7 +55,7 @@ export default function viteJitar(pluginConfig: PluginConfig): PluginOption
     const segments = pluginConfig.segments ?? [];
     const middlewares = pluginConfig.middleware ?? [];
 
-    const scopes = ['shared', ...segments, 'remote'];
+    const scopes = [ ...segments, 'remote'];
```

Comment:
The removal of the `shared` scope creates the situation that common files need to be checked separately. This new situation that isn't covered yet.

For example, if a utility module is used on both the backend and the frontend, the build version will be used on the backend, and the source version on the frontend. In most cases this won't be a problem. But in cases where the imports / exports needed to be rewritten in the utility module, this could lead unexpected behavior.